### DISCUSSION TITLE
OpenAI最終理想是打造財富分配後的烏托邦? 💵

### DISCUSSION THREAD

**Discussion by xxrjun** [1101-GS4719-C/Python-Community-GS4719-C#1203] [General]
The New York Times前幾天採訪OpenAI現任CEO Sam Altman。閱讀完這篇文章後，其中讓我最深刻的無非是末段提到Sam Altman最終的理想。

原文出處: [The ChatGPT King Isn’t Worried, but He Knows You Might Be](https://www.nytimes.com/2023/03/31/technology/sam-altman-open-ai-chatgpt.html)

> "他的宏偉想法是，OpenAI 將通過創建 AGI 獲取世界上大部分的財富，然後將這些財富重新分配給人們。在納帕，當我們坐在他牧場中心的湖邊聊天時，他拋出了幾個數字——1000 億美元、1 萬億美元、100 萬億美元。"

 >"如果 AGI 確實創造了所有這些財富，他不確定公司將如何重新分配這些財富。在這個新世界中，金錢可能意味著非常不同的東西。
> 
> 但正如他曾經告訴我的那樣：“我覺得 AGI 可以在這方面提供幫助。”"

補充: AGI (artificial general intelligence，通用人工智慧)

先說我自己是還蠻喜歡這類型改變人類社會與世界的理想，例如Elon Musk對於上火星的理想，並透過許多方式獲得需要的資金。而Sam Altman目前似乎也在進行類似的大事。

撇開能不能做到，我很好奇如果Sam Altman能夠達成這樣的理想，政府應該扮演甚麼角色，如果一家公司或是企業能夠做到財富重分配，那權力的掌握是否就轉移到公司身上? 這樣又算是資本主義還是社會主義呢? 我也想知道財富和權力掌握在企業手中可能帶來的潛在風險，畢竟，公司的首要目標通常是追求利潤，而不是社會福祉。政府可能都難以建立一個完美有效的監管機制，OpenAI該怎麼做呢?

另外Sam Altman也有提到我心中好奇的點，在這樣的世界中，"金錢究竟意味著甚麼"。財富就算能夠重分配，但有限的資源不可能，也因此財富較多的人能夠掌握更多資源。這樣的條件下有誰能成為更好的分配者。若是財富重分配，又會如何影響人們對於金錢的看法。

總之，我對Sam Altman的理想抱有樂觀的態度，很好奇Sam Altman後續會怎麼做，也好奇大家有什麼想法 :thinking::thinking:

**Comment by DuxtinTai** [comment]
這是一個非常有趣和挑戰性的問題
從政府的角度來看，他們可能需要扮演監管機構的角色，以確保OpenAI運作的透明和公正。如果OpenAI掌握太多的權力，可能會對政府和公民的權力產生負面影響，因此建立一個明確的權力平衡機制是非常重要的。再來，從OpenAI的角度來看，如果公司擁有大部分財富和權力，那麼這可能對公司的利潤追求和社會福祉之間產生衝突。公司的首要目標通常是追求利潤，而不是社會福利。我認為，這種矛盾可能會導致公司不願意投入資源或時間來實現社會福利，想對而言，政府可能難以建立完美有效的監管機制來解決這個問題。
